SourceEditorCompose/src/main/res/xml/backup_rules.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
We probably don't need this file (nor the `xml` folder).

Reply:
I tried deleting the file and the folder, but it looks like it's getting used in debugAndroidManifest.xml for the attribute android:fullBackupContent="@xml/backup_rules"

Will try to figure out how to get past that, but for now resolving this. Thanks